electrum/base_wizard.py

```diff
@@ -492,7 +492,11 @@ def restore_from_seed(self):
     def on_restore_seed(self, seed, is_bip39, is_ext):
         self.seed_type = 'bip39' if is_bip39 else mnemonic.seed_type(seed)
         if self.seed_type == 'bip39':
-            f = lambda passphrase: self.on_restore_bip39(seed, passphrase)
+            def f(passphrase):
+                # TODO: Check with @SomberNight if this is ok
+                self.seed = seed
```

Comment:
@SomberNight is it ok to persist sensitive data on the `BaseWizard` instance like this?

I'm attaching `seed` and `passphrase` to the instance so I can access them later in the derivation dialog.

I'm not exactly sure how long the `BaseWizard` instance lives for, but I assume it's destroyed once the wizard has completed so `seed`/`mnemonic` would be garbage collected.

Reply:
As on IRC, I think (seed, passphrase) could be passed around as parameters. Probably optional keyword-only args, as the hardware wallet codepath will not have them.